src/chromo/models/pythia8.py

```diff
@@ -95,15 +95,30 @@ def __init__(self, evt_kin, *, seed=None, config=None):
         # or init() may fail.
         if "PYTHIA8DATA" in environ:
             del environ["PYTHIA8DATA"]
-        self._pythia = self._lib.Pythia(datdir, True)
+        self._pythia = self._lib.Pythia(datdir, banner)
 
         if config is None:
             self._config = ["SoftQCD:inelastic = on"]
         else:
             self._config = self._parse_config(config)
 
+        # Common settings
+        self._config += [
+            # use our random seed
+            "Random:setSeed = on",
+            # Pythia's RANMAR PRNG accepts only seeds smaller than 900_000_000,
+            # this may change in the future if they switch to a different PRNG
+            f"Random:seed = {self.seed % 900_000_000}",
+            # reduce verbosity
+            "Print:quiet = on",
```

Comment:
Doesn't this always overwrite, these options even if the user doesn't want the quite?
Also, the TODO comment has been abandoned that was mentioning that we want to use our PRNG in future.

Reply:
If you think it is necessary to give users the freedom to override `Print:quiet = on`, go ahead. I thought it is not necessary. Strictly speaking, we only need to make sure that the handling randomness and initial state (beam particles, frame, etc) cannot be overridden by a user.

Reply:
Ok, added possibility to Print:quiet = off